react-frontend/src/components/Guides/CollaborationTools/collaborationTools.js

```diff
@@ -14,641 +15,654 @@ import {
-        <li>
-          Do not click on suspicious links or attachments sent via chat or
-          emails about the platform or tool.
-        </li>
-        <li>
-          Before you upload or share a document, consider whether it is
-          appropriate. For example, documents where the copyright is owned by a
-          third party should not be shared, unless your organization has a
-          license that permits this.
-        </li>
-        <li>
-          Notify participants in advance if you will be recording or
-          transcribing the meeting and manage recordings in accordance with your
-          organization’s policies.
-        </li>
-        <li>
-          Do not use private messaging for confidential information, as hosts
-          may have access to chat logs.
-        </li>
-      </ul>
-      <Title3>
-        <strong>Free Zoom and security </strong>
-      </Title3>
-      <p>
-        During the COVID-19 public health emergency, many of you across the B.C.
-        public sector may be using the free version of Zoom to communicate with
-        colleagues and clients. Please be aware of the privacy and security
-        vulnerabilities – primarily, weak encryption that generally falls short
-        of B.C. government standards. This vulnerability means information is
-        not adequately protected. If you are sharing confidential information,
-        do not use Zoom if you have access to a secure alternative.
-      </p>
-      <p>
-        Other vulnerabilities include the ability for hackers to steal your
-        Windows login credentials, and for uninvited participants to join and
-        disrupt your Zoom meetings – called “Zoombombing.”
-      </p>
-      <p>
-        It is up to each broader public sector entity to determine whether Zoom
-        is appropriate for use within that organization. Broader public sector
-        entities are encouraged to consult with their privacy and security
-        experts and legal advisors to assess the risks associated with their
-        intended use of Zoom. Where a broader public sector entity has
-        determined that the use of Zoom is appropriate within its organization,
-        the following steps can be taken  to improve the security of the free
-        version of Zoom.
-      </p>
-      <p>
-        <strong>General tips</strong>
-      </p>
-      <ul>
-        <li>
-          <strong>Use the web version of Zoom </strong>on your desktop or
-          laptop.
-        </li>
-        <li>
-          <strong>Ensure the Zoom client app is the current version</strong> and
-          regularly check for updates. For more information on installing apps,
-          read the{' '}
-          <a href="https://www2.gov.bc.ca/gov/content/governments/services-for-government/policies-procedures/appropriate-use-policy">
-            Applications and Software Guide
+    <DocumentTitle title="Digital Principles - Digital Government - Province of British Columbia">
```

Comment:
CollaborationTools.js has title as Digital Principles, expected?

Reply:
Oops ... no. Sorry for not catching before you merged. I will file a separate issue and PR for this.